pkg/apis/triggers/v1beta1/event_listener_types.go

```diff
@@ -64,6 +64,7 @@ type EventListenerSpec struct {
 	NamespaceSelector NamespaceSelector           `json:"namespaceSelector,omitempty"`
 	LabelSelector     *metav1.LabelSelector       `json:"labelSelector,omitempty"`
 	Resources         Resources                   `json:"resources,omitempty"`
+	TargetURI         string                      `json:"target-uri,omitempty"`
```

Comment:
Can you update in eventlistener doc about `TargetURI` with example

Reply:
String is fine for optional. I changed it to `CloudEventURI`.